src/containers/Teams/index.tsx

```diff
@@ -1,22 +1,78 @@
 import React, {FC, ReactNode, useCallback, useEffect, useState} from 'react';
-import {useHistory, useParams} from 'react-router-dom';
+import {Link, useHistory, useParams} from 'react-router-dom';
 
-import {BreadcrumbMenu, EmptyPage, FlatNavLinks, PageTitle} from 'components';
+import {BreadcrumbMenu, EmptyPage, FlatNavLinks, Icon, IconType, PageTitle} from 'components';
 import {TEAMS} from 'constants/teams';
-import {TeamMember, TeamName, TeamsUrlParams} from 'types/teams';
+import {PageDataObject} from 'types/page-data';
+import {TeamMember, TeamName, TeamsUrlParams, TeamTabOptions} from 'types/teams';
 import {getTeamMembers} from 'utils/data';
 
+import InternalHowToSetUpPaymentBoard from './Resources/InternalHowToSetUpPaymentBoard';
+import InternalNewUserOperations from './Resources/InternalNewUserOperations';
+import InternalProductDevelopment from './Resources/InternalProductDevelopment';
+import InternalTeamLeadOnboarding from './Resources/InternalTeamLeadOnboarding';
 import TeamMemberCard from './TeamMemberCard';
+import TeamOverview from './TeamOverview';
+import TeamTabs from './TeamTabs';
 import './Teams.scss';
 
 const teamMembers = getTeamMembers();
 
+const sortTeamMembers = (members: TeamMember[]): TeamMember[] => {
+  const teamLeads = members
+    .filter(({isLead}) => isLead)
+    .sort((member1, member2) => (member1.displayName > member2.displayName ? 1 : -1));
+  const otherMembers = members
+    .filter(({isLead}) => !isLead)
+    .sort((member1, member2) => (member1.displayName > member2.displayName ? 1 : -1));
+  return teamLeads.concat(otherMembers);
+};
+
+const pageData: PageDataObject = {
+  'how-to-set-up-payment-board': {
+    content: <InternalHowToSetUpPaymentBoard />,
+    name: 'How to set up payment boards',
+  },
+  'new-user-operations': {
+    content: <InternalNewUserOperations />,
+    name: 'How to onboard new users',
+  },
+  'product-development': {
+    content: <InternalProductDevelopment />,
+    name: 'How the product development process works',
+  },
+  'team-lead-onboarding': {
+    content: <InternalTeamLeadOnboarding />,
+    name: 'How to onboard team leads',
+  },
+};
+
 const Teams: FC = () => {
   const history = useHistory();
-  const {team: teamParam} = useParams<TeamsUrlParams>();
+  const {team: teamParam, tab: tabParam, resource: resourceParam} = useParams<TeamsUrlParams>();
   const [filteredMembers, setFilteredMembers] = useState<TeamMember[]>(teamMembers);
   const [teamFilter, setTeamFilter] = useState<TeamName>(TeamName.all);
 
+  useEffect(() => {
+    if (teamParam && tabParam) {
+      if (teamParam === TeamName.all) {
+        if (!(tabParam === TeamTabOptions.members || tabParam === TeamTabOptions.resources)) {
+          history.replace(`/teams`);
+        }
+      } else if (teamParam !== TeamName.all) {
```

Comment:
should this just be `else`?

Reply:
Actually, I put `else` there but it complained about `no-lonely-if` as it has one `if` statement inside it. So, had to add this. :(